.github/workflows/release.yml

```diff
@@ -109,9 +116,23 @@ jobs:
           # fi
 
           if [[ ${input_holochain_source_branch} != "" ]]; then
```

Comment:
I find it easier to read with a positive condition and then `else`

Reply:
that's fair. i'm not really motivated to do a survey and see what the majority would find more readable. so i'll leave this at subjective preference and will marked the comment as resolved :sweat_smile: